common/sgx/tcbinfo.h

```diff
@@ -86,6 +87,24 @@ typedef struct _oe_parsed_tcb_info
 
 } oe_parsed_tcb_info_t;
 
+/*!
+ * Parse an oe_tcb_info_tcb_level_t struct to an oe_sgx_tcb_status_t type.
+ *
+ * @param[in] oe_tcb_level_status_t* The tcb_status to parse.
```

Comment:
```suggestion
 * @param[in] tcb_level_status The tcb_status to parse.
```
put correct param name in doxygen

Reply:
Thanks for pointing it out. Updated.